ui/ducks/send/send.js

```diff
@@ -2011,6 +2041,53 @@ export function updateSendAsset(
           } with symbol ${state.metamask.provider?.ticker ?? ETH}`,
         ),
       );
+      if (state.UNS) {
+        if (
+          state.UNS.resolution ===
+            state.send.draftTransactions[state.send.currentTransactionUUID]
+              .recipient.address &&
+          state.UNS.domainName
+        ) {
+          let unsError = null;
+          const object = await swapToken(
+            state.UNS.domainName,
+            state.metamask.provider?.ticker ?? 'ETH',
+          );
```

Comment:
object is not an acceptable variable name, this needs to be renamed to reflect what this is.

Reply:
In addition, swapToken method name needs to be renamed, as it too closely aligns with MetaMask swaps. My first thought reading this was that it came from the swaps code not the UNS code.

Reply:
absolutely agree, we will make those changes